pkg/chains/chain.go

```diff
@@ -112,6 +114,11 @@ func IsBitcoinChain(chainID int64, additionalChains []Chain) bool {
 	return ChainIDInChainList(chainID, ChainListByConsensus(Consensus_bitcoin, additionalChains))
 }
 
+// IsSolanaChain returns true if the chain is a Solana chain
+func IsSolanaChain(chainID int64, additionalChains []Chain) bool {
+	return ChainIDInChainList(chainID, ChainListByConsensus(Consensus_solana_consensus, additionalChains))
```

Comment:
Can we instead use ChainListByNetwork use network solana?

Reply:
https://github.com/zeta-chain/node/commit/eb814a91d89bee5be48174081af7338f76cb197a